### DISCUSSION TITLE
How to weed out some 'users'

### DISCUSSION THREAD

**Discussion by ellingmel** [microsoft-search/pnp-modern-search#2960] [Q&A]
So I am using * in the default value of the search results - I want to show all employees before a search and then allow the user to search and / or filter. The problem is I have some temporary users, some operational users etc plus I see that groups and rooms etc are included in that inital search. Is there a way I can initially filter? Either using that default value or Query Text or Refinement filters?

**Comment by kasperbolarsen** [comment]
The company I am working for have published our guidelines for cleaning up the people search :-) https://docs.intraactive.net/?doc=/intraactive-intranet/phonebook-results/
Hopefully you can find something of value here.
